scripts/migrate-auth0-users.js

```diff
@@ -20,17 +20,25 @@
  *   "clientId": "SECRET"
  * }
  *
+ * If deleting existing users in the to tenant id desired, the connection id is
+ * also required. The connection id typically is only discoverd by opening up
+ * an API explorer and listing all connections.
+ *
```

Comment:
I feel like this belongs above the example format for `client.json` in a parenthetical. Also, I think this needs a bit more information about how to find the `connectionId` and it should be listed in the example base format (probably with a single line comment saying it's only required for user deletion).